gulpfile.babel.js

```diff
@@ -136,51 +153,72 @@ const testSass = () =>
 
 export const validateSass = gulp.parallel(lintSass, testSass);
 
-const cleanCss = () => del(config.files.src.css);
+const cleanCss = () => del(config.files.build.css);
 
 const compileCss = () =>
   gulp.src(config.files.src.sassEntryPoint)
     .pipe(sourcemaps.init())
     .pipe(sassGlob())
     .pipe(sass(config.sass.options).on('error', sass.logError))
-    .pipe(replace(/\.\.\/\.\.\/fonts\//g, '../fonts/'))
-    .pipe(rename(config.files.out.cssFilename))
+    .pipe(rename(config.files.build.cssFilename))
     .pipe(sourcemaps.write('./'))
-    .pipe(gulp.dest(config.dir.src.css));
+    .pipe(gulp.dest(config.dir.build.css));
 
 export const generateCss = gulp.series(cleanCss, compileCss);
 
-export const build = gulp.parallel(validateSass, generateCss);
+export const buildCss = gulp.parallel(validateSass, generateCss);
+
+// Patterns tasks
+
+const cleanLinks = () => del([config.dir.build.meta, config.dir.build.patterns]);
+
+const linkMeta = () =>
+  gulp.src(config.dir.src.meta + '/**/*')
+    .pipe(gulp.dest(config.dir.build.meta));
+
+const linkPatterns = () =>
+  gulp.src(config.dir.src.patterns + '/**/*')
+    .pipe(gulp.dest(config.dir.build.patterns));
```

Comment:
Originally though about symlink these, decided against to save having to mount `source` into `watch-build` as well.